assets/app/views/directives/delete-button.html

```diff
@@ -0,0 +1,8 @@
+<div class="actions action-button pull-right">
+	<i class="fa fa-trash-o fa-2x" 
```

Comment:
take the fa-2x off here since its getting set by ng-class so there is no confusion later